docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -98,6 +98,10 @@ documentation][scale].
   that the controller will spend on draining a control plane node.
   The default value is 0, meaning that the node can be drained without any time limitations.
 
+* `machineTemplate.nodeVolumeDetachTimeout` - is a *metav1.Duration defining how long the controller
```

Comment:
This change to the contract for control plane provider should be documented as a breaking change in https://main.cluster-api.sigs.k8s.io/developer/providers/v1.2-to-v1.3.html; this is an example from the v1.2 breaking changes for providers https://main.cluster-api.sigs.k8s.io/developer/providers/v1.1-to-v1.2.html#required-api-changes-for-providers

Reply:
@fabriziopandini hi, thanks for the review! This has been documented now in the providers' migration guide as you suggested. Additionally, I also noted a newly introduced annotation `machine.cluster.x-k8s.io/exclude-wait-for-node-volume-detach` in the same document for breaking changes, hope it makes sense. Let me know what you think and if it needs to be moved/removed, I just wanted to make sure we are not leaving it without documenting it anywhere. Thanks a lot!